glide.yaml

```diff
@@ -7,17 +7,17 @@ import:
 - package: github.com/gorilla/mux
   version: ^1.6.1
 - package: k8s.io/client-go
-  version: v6.0.0
+  version: v8.0.0
 - package: k8s.io/apimachinery
-  version: release-1.9
+  version: kubernetes-1.11.1
 - package: github.com/golang/protobuf
-  version: v1.2.0
+  version: b4deda0973fb4c70b50d226b1af49f3da59f5265
```

Comment:
Make sure this is compatible with the Prometheus client. I had to change this to specifically use v1.2.0 because earlier versions could not work with it.

@lucasponce how does this work with my PR #598 ? Are you able to apply that PR to this one and see that it can compile and tests pass? (just need to `make build test` after applying that PR to this one - if it passes tests, everything should be good).

Reply:
I think this is newer version. It throwed a warning that v1.2.0 was too old for the client and this was the target version needed.
I'll test it in ~1.5h

Reply:
I've bumped back profobuf to v1.2.0.
There is a warning:
```
[WARN]	Conflict: github.com/golang/protobuf rev is currently v1.2.0, but k8s.io/api wants b4deda0973fb4c70b50d226b1af49f3da59f5265
```
But kiali log is clean and I guess it will be similar having client-go v6.0.0 + protobuf v1.2.0 as client-go v8.0.0 + protobuf v1.2.0, so if no suspicious problem I guess it's fine.
Going to rebase with your #598 and check if it compiles.

Reply:
I fetch #598 locally, rebased with this PR.
I needed to run a `make dep-update` and update in the rebased branch these libraries:
```
$ git status
On branch pr598
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   glide.lock

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	vendor/github.com/beorn7/
	vendor/github.com/matttproud/
	vendor/github.com/prometheus/client_model/
	vendor/github.com/prometheus/procfs/
```
But after this step, a `make build test` works fine.

So not these missing dependencies should be part of this PR or a different one more close to 598 ?

Reply:
Since you confirmed that all of these dependencies work with 598, it is fine to include it in this PR now. We do not need to create a second PR more close to 598 - I plan on getting 598 merged on Monday anyway. I will wait to merge 598 until you get this PR merged. If any issues  come up we can address in another dependency PR. But for now, having just this one PR is OK in my opinion since you can see 598 working with everything.